scripts/release-docker.sh

```diff
@@ -77,6 +77,10 @@ mina-rosetta)
 mina-rosetta-ubuntu)
   DOCKERFILE_PATH="dockerfiles/stages/1-build-deps dockerfiles/stages/2-toolchain dockerfiles/stages/3-opam-deps dockerfiles/stages/4-builder dockerfiles/stages/5-prod-ubuntu"
   ;;
+mina-snapp-test-transaction)
+  DOCKERFILE_PATH="dockerfiles/Dockerfile-snapp-test-transaction"
+  DOCKER_CONTEXT="dockerfiles/"
```

Comment:
This container doesn't actually require a context if you remove the unused entrypoint script